src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs

```diff
@@ -175,7 +175,7 @@ private void LogRequestStarting(HostingApplication.Context context)
 
             _logger.Log(
                 logLevel: LogLevel.Information,
-                eventId: LoggerEventIds.RequestStarting,
+                eventId: new EventId(LoggerEventIds.RequestStarting, "RequestStarting"),
```

Comment:
I think you could go to `LoggerEventIds` and change them from ints to `EventId` structs (you'd also have to change from `const` to `static readonly` fields but it's an internal type so I think that's OK. Could you give that a look @mikeo52 ?

Reply:
@anurse I think I've got the change you were after made in this regard.  Take a look and let me know.  I'm having trouble building locally since my local setup's a little borked, so my apologies if the extra commits are making noise.